resources/views/emails/auth/reset_password_successfull.blade.php

```diff
@@ -3,24 +3,14 @@
 @section('content')
     <table border="0" cellpadding="0" cellspacing="0" role="presentation" style="vertical-align:middle;" width="100%">
         <tbody>
-        <tr>
-            <td align="center" style="font-size:0px;padding:10px 25px;padding-right:25px;padding-left:25px;word-break:break-word;">
-                <div style="font-family:open Sans Helvetica, Arial, sans-serif;font-size:16px;line-height:1;text-align:center;color:#000000;">Dear <a href="#" style="text-decoration:none !important;color:black !important; cursor:default !important">{!!$user_email!!}</a>,</div>
-            </td>
-        </tr>
         <tr>
             <td align="center" style="font-size:0px;padding:10px 25px;word-break:break-word;">
-                <div style="font-family:open Sans Helvetica, Arial, sans-serif;font-size:16px;line-height:1;text-align:center;color:#000000;">You are receiving this email because you changed your password.</div>
+                <div style="font-family:open Sans Helvetica, Arial, sans-serif;font-size:16px;line-height:1;text-align:center;color:#000000;">The password was reset for FNid <a href="#" style="text-decoration:none !important;color:black !important; cursor:default !important; font-weight:bold">{!!$user_email!!}</a>.</div>
```

Comment:
please dont explicitly mention **FNid** instead use {!! Config::get("app.name") !!}

Reply:
this has been fixed